index.mako

```diff
@@ -758,25 +758,36 @@ Install the pre-commit script.
 
 Options:
 
-- `-f`, `--overwrite`: overwrite existing hooks / remove migration mode.
-- `--install-hooks`: also install hook environments for all available hooks.
+- `-f`, `--overwrite`: Replace any existing git hooks with the pre-commit
+  script.
+- `--install-hooks`: Also install environments for all available hooks now
+  (rather than when they are first executed). See [`pre-commit
+  install-hooks`](#pre-commit-install-hooks).
 - `-t {pre-commit,pre-push,prepare-commit-msg,commit-msg}`,
   `--hook-type {pre-commit,pre-push,prepare-commit-msg,commit-msg}`:
-  which hook type to install.
-- `--allow-missing-config`: whether to allow the installed hook scripts to
-  permit a missing configuration file.
+  Specify which hook type to install.
+- `--allow-missing-config`: Hook scripts will permit a missing configuration
+  file.
```

Comment:
would you like to backport these changes to the help text? (iirc these are grafted directly from there)

Reply:
Sure, can you point me in that direction? This was my first encounter with `.mako` files

Reply:
ah I mean in the code :) https://github.com/pre-commit/pre-commit/blob/master/pre_commit/main.py

Reply:
Oh, sure